### PR TITLE
getcap: add page

### PR DESCRIPTION
**Pull request by saikatsgupta**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by saikatsgupta** [comment]
Closes #8523

**Review by kbdharun — COMMENTED** [review]
The page LGTM, We generally prefer `manned.org` (https://manned.org/getcap.8) when there isn't developer documentation or an official man page. More info on [contributing guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links).

But in your case, the other manual page has detailed information. Let me ask somebody else for their opinion.

File: pages/linux/getcap.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# getcap
+
+> Command to display the name and capabilities of each specified file.
+> More information: <https://www.man7.org/linux/man-pages/man8/getcap.8.html>.
```
Comment (CleanMachine1) [inline pages/linux/getcap.md:4]:
```suggestion
> More information: <https://manned.org/getcap>.
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by Nico385412 — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

File: pages/linux/getcap.md (hunk 1 above)
Comment (kbdharun) [reply pages/linux/getcap.md:4]: @saikatsgupta Can you commit this change?
 This PR is ready to be merged after this.

**Comment by prateeknischal** [comment]
Probably should also add the `-r` option to evaluate all the files recursively. For example, 
```bash
$ getcap -r /usr/bin
/usr/bin/mtr-packet cap_net_raw=ep
/usr/bin/ping cap_net_raw=ep
```

**Comment by prateeknischal** [comment]
> > Probably should also add the `-r` option to evaluate all the files recursively. For example,
> > ```shell
> > $ getcap -r /usr/bin
> > /usr/bin/mtr-packet cap_net_raw=ep
> > /usr/bin/ping cap_net_raw=ep
> > ```
> 
> Thanks for the suggestion, Can you create a PR to add this option?

Done! #8691

**Pull request merged**